webpack.config.cjs

```diff
@@ -41,7 +41,6 @@ module.exports = function (env, argv) {
           test: /\.(ts|tsx|js|jsx)$/,
           exclude: /node_modules/,
           use: [
-            isDevelopment && 'coverage-istanbul-loader',
```

Comment:
Can we change this to `!isDevelopment && 'coverage-istanbul-loader'` instead of removing it entirely? I think removing this loader completely would prevent us from tracking coverage for E2E tests.
The cypress config file only uses this loader for component tests and not E2E tests.

Reply:
The line you linked above is under `component: ` so it only applies to component tests. We should be able to collect coverage from E2E tests as well.

Reply:
Updated with `isProduction` and a clarifying comment in case we're wrong :)